.github/workflows/build_and_deploy.yml

```diff
@@ -50,19 +50,6 @@ jobs:
           azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
           pull-request-number: ${{ github.event.number }}
 
-      - uses: azure/login@v2
-        with:
-          creds: ${{ secrets.AZURE_CREDENTIALS }}
-
-      - name: Seed Review App
```

Comment:
Why remove db:seed?

Reply:
@saliceti see this commit message https://github.com/DFE-Digital/itt-mentor-services/pull/893/commits/c78dea254a580f06fc44328c60a3e1c2924d2e1c

TL;DR: `rails db:prepare` automatically runs seeds for you when initialising a new database.